src/Cake.Common/Tools/NuGet/Pack/NuGetPacker.cs

```diff
@@ -214,6 +214,12 @@ private ProcessArgumentBuilder GetArguments(FilePath filePath, NuGetPackSettings
                     settings.Properties.Select(property => string.Concat(property.Key, "=", property.Value))));
             }
 
+            // Keep temporarily autogenerated NuSpec file?
+            if (settings.KeepTemporaryNuSpecFile)
+            {
+                builder.Append("-KeepTemporaryNuSpecFile");
```

Comment:
@asbjornu is there a reason that this is getting appended? It doesn't seem to be getting used anywhere.